src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -64,12 +64,14 @@ DlgPrefInterface::DlgPrefInterface(QWidget * parent, MixxxMainWindow * mixxx,
     ComboBoxLocale->model()->sort(0); // Sort languages list
     ComboBoxLocale->insertItem(0, "System", ""); // System default locale - insert at the top
 
-    //
     // Skin configurations
-    //
-    QString warningString = "<img src=\":/images/preferences/ic_preferences_warning.png\") width=16 height=16 />"
-        + tr("The minimum size of the selected skin is bigger than your screen resolution.");
-    warningLabel->setText(warningString);
+    QIcon sizeWarningIcon(":/images/preferences/ic_preferences_warning.svg");
+    QString sizeWarningString =
+            "<img src=\":/images/preferences/ic_preferences_warning.svg\" "
+            "width=16 height=16/>  " +
+            tr("The recommended minimum size of the selected skin is bigger than your "
```

Comment:
Beta usually implies string freeze. Should we include this clarification anyway?

Reply:
Oops, I forgot. Let's revert (and maybe l add it for 2.4) then.

Reply:
@ronso0 Then let's split the wording change from the rest of this PR and afterward merging file a separate PR for main.

Reply:
done. clang-format changed it but the tr string is the same.